Lib/test/test_named_expression.py

```diff
@@ -394,21 +394,21 @@ def test_named_expression_scope_10(self):
         self.assertEqual(b, [1, 1])
         self.assertEqual(a, 1)
 
-    # the following test is not from CPyrgon and just as refernce for a common scoping problem of RustPython
-    @unittest.skip # needs skipping due to weired behaviour
+    # the following test is not from CPython and just as refernce for a common scoping problem of RustPython
     def test_named_expression_scop_10_rp_scope_prob(self):
         def foo():
-            rr=0
+            rr=36
```

Comment:
is this changed by following original test code or original fix?

Reply:
Thought, this is no CPython test, I added this test as reference for the existing scope issue (see comment to the test case). In the old verison it only tested, that data is propagated out of the inner scope. The new version tests also that data is also propagated to the inner scope. The snippets would maybe a better palce for own test cases, but as reference, until completion of the named expressions, I keep it here.

Reply:
will you please move it to snippets if there is no reason to keep it in unittest directory?